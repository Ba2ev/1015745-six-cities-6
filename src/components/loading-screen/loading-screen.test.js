import React from 'react';
import {render} from '@testing-library/react';
import LoadingScreen from './loading-screen';

it(`LoadingScreen should render correctly`, () => {
  const {getByText} = render(<LoadingScreen />);

  const mainText = getByText(`Loading ...`);

  expect(mainText).toBeInTheDocument();
});

it(`LoadingScreen should render snapshot correctly`, () => {
  const {container} = render(<LoadingScreen />);

  expect(container).toMatchSnapshot();
});
