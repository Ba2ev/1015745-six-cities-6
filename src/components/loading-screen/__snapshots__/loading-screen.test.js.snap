// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingScreen should render snapshot correctly 1`] = `
<div>
  <p>
    Loading ...
  </p>
</div>
`;
